### PR TITLE
Remove unwanted steps from Tremor activity

### PR DESCRIPTION
**Pull request by Erin-Mounts**
per Andrew, should only include lap, extended at shoulder height, and touching nose.

Also, since we don't yet know enough to create a score for Tremor, remove it from the dashboard for now (again per Andrew). A future release of mPower will add it back once we can analyze enough data to come up with a way to score it.

### CODE REVIEW

**Comment by syoung-smallwisdom** [comment]
Since we are digging in and making changes to RK already (including adding this module) could we instead architect this with an enum for defining which tremor steps should be included?

**Comment by Erin-Mounts** [comment]
good point, that's a much better way to handle it. will update this and submit a parallel RK PR.

File: mPowerSDK/TasksAndSteps/APHActivityManager.m [hunk 1]
```diff
@@ -93,8 +93,14 @@
...
-static NSString *const kTremorAssessmentTitleIdentifier     = @"Tremor Activity";
-static NSTimeInterval kTremorAssessmentStepDuration         = 10.0;
+static NSString *const kTremorElbowBentInstructionStepIdentifier    = @"instruction5";
+static NSString *const kTremorElbowBentCountdownStepIdentifier      = @"countdown3";
+static NSString *const kTremorElbowBentStepIdentifier               = @"tremor.handAtShoulderLengthWithElbowBent";
+static NSString *const kTremorQueenWaveInstructionStepIdentifier    = @"instruction7";
+static NSString *const kTremorQueenWaveCountdownStepIdentifier      = @"countdown5";
+static NSString *const kTremorQueenWaveStepIdentifier               = @"tremor.handQueenWave";
+static NSString *const kTremorAssessmentTitleIdentifier             = @"Tremor Activity";
+static NSTimeInterval kTremorAssessmentStepDuration                 = 10.0;
```
Comment (syoung-smallwisdom) [inline mPowerSDK/TasksAndSteps/APHActivityManager.m]: Still needed?

**Comment by syoung-smallwisdom** [comment]
One nit. Otherwise, looks good.

**Pull request merged**